homepage.html

```diff
@@ -12,11 +12,46 @@
         h1
         {
             position: relative;
-            top: 50%;
+            top: 30%;
```

Comment:
Put all the style code into a "Style" folder and make reference from there. Study about the methods to implement css codes.